Dockerfile

```diff
@@ -0,0 +1,79 @@
+# Install dependencies only when needed
+FROM node:14 AS deps
+
+ARG TWOOPSTRACKER_API_URL
+ARG NEXT_PUBLIC_TWOOPSTRACKER_API_URL
```

Comment:
Ola @kilemensi 
Seems like it's used to generate the `BASE_URL` [here](https://github.com/CodeForAfrica/TwoopsTrackerUI/blob/main/src/lib/index.js#L5)
Why was it needed in the first place and should I update the `BASE_URL` to use the `TWOOPSTRACKER_API_URL` ?

Reply:
I believe `TWOOPSTRACKER_API_URL` and `NEXT_PUBLIC_TWOOPSTRACKER_API_URL` point to the same back-end instance & hence we should only use one @esirK 

Since it's not needed on the FE, we can drop `NEXT_PUBLIC_TWOOPSTRACKER_API_URL`